fusor-ember-cli/app/mixins/validated-input-mixin.js

```diff
@@ -49,6 +49,12 @@ export default Ember.Mixin.create({
       this.set("showValidationError", true);
     },
 
+    showValidationErrorsKeyUp() {
```

Comment:
Can we add a comment here explaining the desired behavior? It's difficult to understand the intent behind this boolean expression.